docs/crd.md

```diff
@@ -219,6 +221,7 @@ FlinkCluster
       * **jobManagerService**: The status of the JobManager service.
         * **name**: The resource name of the JobManager service.
         * **state**: The state of the JobManager service.
+        * *nodePort*: Accessible JobManager service through NodePort 
```

Comment:
Can we update it to: **nodePort** (optional): The node port, present when `accessScope` is `NodePort`